scripts/start_server.sh

```diff
@@ -4,14 +4,14 @@
 #
 # Copyright 2017 Eric Bidelman <ericbidelman@chromium.org>
 
-export PYTHONPATH=cs-env/lib/python3.10/site-packages:$PYTHONPATH
+export PYTHONPATH=cs-env/lib/python3/site-packages:$PYTHONPATH
 export GOOGLE_CLOUD_PROJECT='cr-status-staging'
 export SERVER_SOFTWARE='gunicorn'
 export GAE_ENV='localdev'
 export DATASTORE_EMULATOR_HOST='localhost:15606'
 
 
-gunicorn --bind :8080 --workers 4 main:app
+gunicorn --bind :8080 --workers 1 main:app
```

Comment:
Is this still needed?  It's a difference running locally and how the app runs on GAE.

Reply:
This can be reverted